09-MasterDetail/src/PrismSample/ViewModels/TabAPageViewModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Prism.Mvvm;
+
+namespace PrismSample.ViewModels
+{
+    public class TabAPageViewModel : BindableBase
+    {
+        public TabAPageViewModel()
+        {
+
+        }
+    }
+}
```

Comment:
Implement IInitialize... and pull from the NavigationParameters a message for this tab...